moveit_ros/moveit_servo/include/moveit_servo/servo_parameters.h

```diff
@@ -89,4 +89,46 @@ struct ServoParameters
 
 using ServoParametersPtr = std::shared_ptr<ServoParameters>;
 
+template <typename T>
```

Comment:
I'm not sure this function is supposed to be exposed since it's only used inside `servo_parameters.cpp` which is not meant to be a generic parameter utility library. If you are looking for something similar, maybe consider https://github.com/PickNikRobotics/rosparam_shortcuts/pull/15.

Reply:
I agree. `declareOrGetParam` should not be exported in this case.

I will switch to just move `servo_parameters.cpp` to `src` in order to fix https://github.com/ros-planning/moveit2/issues/306.

Reply:
If you move this function definition to servo_parameters.cpp so it's not exposed as Henning said, I would approve the PR. This looks like a nice step forward and a fix to #306.